### ISSUE TITLE
OTA targeted images not installed

### ISSUE THREAD

**Issue by mscbpi** [MicrosoftDocs/azure-sphere-issues#74]
Following the guide, I can't get OTA download of the app.

The device is correctly connected to WIFI, configured in device group, subscribed to list:

`azsphere device image list-targeted
Successfully retrieved the current image set for device with ID
'0F8ADE345823314A1EE33DBBF6CEC59C6E15B81E5DA8E3ED9AFC05DD9981BA6CD0C9E397E3CC7756B69D456A9EFC9CAEF78BDB915413EDEB07604DDE88CF63BE' from your Azure Sphere tenant:
 --> Image set ID:   c6c29e8b-e7bf-42cc-a01a-35e57abf9b95
 --> Image set name: ImageSet-Mt3620Blink1-2019.01.30-11.08.10+01:00
Images to be installed:
 --> Mt3620Blink1
   --> Image type:   Application
   --> Component ID: ea976d20-33ad-4453-93fe-7d2af0e0ccc9
   --> Image ID:     eef610fc-d4c6-49b4-a12e-6668b6748490
Command completed successfully in 00:00:02.8656254.`

After multiple reset and 24-hour wait:

`azsphere device image list-installed
No installed images.
Command completed successfully in 00:00:01.3568691.`

**Comment by sergaz-msft** [comment]
@mscbpi thank you for your feedback. Can you please clarify what is the doc you are following?

Thanks!

**Comment by sergaz-msft** [comment]
Since we have not heard back from you we will now proceed to close this thread. If there are further questions regarding this matter, please tag me in your reply. We will gladly continue the discussion and we will reopen the issue.

**Comment by MarekLani** [comment]
@sergaz-msft  can you please reopen this issue, as I am facing same problem. I followed this article: https://docs.microsoft.com/en-us/azure-sphere/quickstarts/qs-first-deployment with one difference, I was using app which uses beta apis, so when creating feed I referenced Retail Evaluation Azure Sphere OS as dependentfeedid (ofc by its id).

**Comment by sergaz-msft** [comment]
@MarekLani can I kindly ask you to open a new issue under the documentation you refer? Though if you are not following the doc as is the best is that a new thread on MSDN is open: https://social.msdn.microsoft.com/Forums/en-US/home?forum=azuresphere

Thank you!

**Comment by MarekLani** [comment]
thanks @sergaz-msft . I have just found out it was network related issue (most probably my internet provider was filtering the communication). Device however was not showing any sign of existing OTA update and deploy problem, which was misleading and so I have created following user voice feedback: https://feedback.azure.com/forums/915433-azure-sphere/suggestions/38281627-provide-ota-logs

**Comment by sergaz-msft** [comment]
Thank you! We will maintain user voice feedback and update accordingly! Glad that you are now unblocked :). Happy coding!

**Comment by Asvaldrz** [comment]
> thanks @sergaz-msft . I have just found out it was network related issue (most probably my internet provider was filtering the communication). Device however was not showing any sign of existing OTA update and deploy problem, which was misleading and so I have created following user voice feedback: https://feedback.azure.com/forums/915433-azure-sphere/suggestions/38281627-provide-ota-logs

Hey @MarekLani, what were the network issues in the end? I have a similar problem at the moment. I can see images targetted, but none will install.
Thanks in advance

**Comment by MarekLani** [comment]
hi @Asvaldrz  if I remember correctly I was connecting the device thru my tethered phone network. When I switched to my home wifi it worked.

**Comment by Asvaldrz** [comment]
@MarekLani  Thanks you for the response. I'm already on a home setup. Thanks again
